app/src/main/java/com/besome/sketch/editor/PropertyActivity.java

```diff
@@ -0,0 +1,358 @@
+        this.d.send(new HitBuilders.ScreenViewBuilder().build());
+    }
+
+    @Override
+    // androidx.core.app.ComponentActivity, androidx.appcompat.app.AppCompatActivity, androidx.fragment.app.FragmentActivity
+    public void onSaveInstanceState(Bundle bundle) {
+        bundle.putString("sc_id", this.q);
+        bundle.putParcelable("project_file", this.n);
+        bundle.putParcelable("bean", this.o);
+        super.onSaveInstanceState(bundle);
+    }
+
+    public void p() {
+        Intent intent = new Intent(getApplicationContext(), ManageImageActivity.class);
+        intent.setFlags(536870912);
+        intent.putExtra("sc_id", this.q);
+        startActivityForResult(intent, 209);
+    }
+
+    class a extends RecyclerView.a<PropertyActivity.a.View$OnClickListenerC0029a> {
+
+        public a() {
+        }
+
+        /* renamed from: a */
+        public void b(View$OnClickListenerC0029a aVar, int i) {
+            ((Integer) PropertyActivity.this.s.get(i)).intValue();
+            if (PropertyActivity.this.v == i) {
+                ef a2 = Ze.a(aVar.t);
+                a2.c(1.1f);
+                a2.d(1.1f);
+                a2.a(300L);
+                a2.a(new AccelerateInterpolator());
+                a2.c();
+                ef a3 = Ze.a(aVar.t);
+                a3.c(1.1f);
+                a3.d(1.1f);
+                a3.a(300L);
+                a3.a(new AccelerateInterpolator());
+                a3.c();
+                aVar.v.setVisibility(0);
+                ColorMatrix colorMatrix = new ColorMatrix();
+                colorMatrix.setSaturation(1.0f);
+                aVar.t.setColorFilter(new ColorMatrixColorFilter(colorMatrix));
+                return;
+            }
+            ef a4 = Ze.a(aVar.t);
+            a4.c(1.0f);
+            a4.d(1.0f);
+            a4.a(300L);
+            a4.a(new DecelerateInterpolator());
+            a4.c();
+            ef a5 = Ze.a(aVar.t);
+            a5.c(1.0f);
+            a5.d(1.0f);
+            a5.a(300L);
+            a5.a(new DecelerateInterpolator());
+            a5.c();
+            aVar.v.setVisibility(8);
+            ColorMatrix colorMatrix2 = new ColorMatrix();
+            colorMatrix2.setSaturation(0.0f);
+            aVar.t.setColorFilter(new ColorMatrixColorFilter(colorMatrix2));
+        }
+
+        @Override // androidx.recyclerview.widget.RecyclerView.a
+        public View$OnClickListenerC0029a b(ViewGroup viewGroup, int i) {
+            return new View$OnClickListenerC0029a(LayoutInflater.from(viewGroup.getContext()).inflate(2131427377, viewGroup, false));
+        }
+
+        @Override // androidx.recyclerview.widget.RecyclerView.a
+        public int a() {
+            return PropertyActivity.this.s.size();
+        }
+
+        /* renamed from: com.besome.sketch.editor.PropertyActivity$a$a  reason: collision with other inner class name */
+        class View$OnClickListenerC0029a extends RecyclerView.v implements View.OnClickListener {
```

Comment:
Could you rename this class to something like `ViewHolder`? Its current name is pretty long and confusing.